src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -297,8 +292,7 @@ def __init__(self, **_data):
         self.text = _data.get('command', '') if _data.get('command', '') else self.text
 
         self.long_summary = _data.get('description', '')
-        self.min_profile = _data.get('min_profile', '')
-        self.max_profile = _data.get('max_profile', '')
+        self.supported_profiles = _data.get('supported_profiles', '')
```

Comment:
I don't think this field is used. Should we remove it or leave it? I believe it is only used in test_help.py

Reply:
You can remove it and see what happens. However, that means test_help needs to be updated to use whatever the right thing is.

Reply:
Yes test_help only uses it in unit tests I added. I was simply verifying that profile information is loaded properly from data. Which we didn't do previously.